homeassistant/components/filter/sensor.py

```diff
@@ -178,16 +179,20 @@ def __init__(self, name, entity_id, filters):
         self._state = None
         self._filters = filters
         self._icon = None
+        self._device_class = None
 
     @callback
     def _update_filter_sensor_state_event(self, event):
         """Handle device state changes."""
+        _LOGGER.debug("Update filter on event: %s", event)
```

Comment:
```suggestion
```
Not sure this is needed. The state change that results should be logged anyway.

Reply:
There is another PR open https://github.com/home-assistant/core/pull/43244 that could benefit from this log :)